### PR TITLE
First version of Apollo API

### PR DESCRIPTION
**Pull request by tmeasday**
Working on transitioning Todos to Apollo for https://github.com/meteor/guide/pull/521

Follow progress on the devlog: https://github.com/meteor/guide/blob/apollo-guide-todos-devlog/content/apollo.md#migration-log

### CODE REVIEW

**Comment by tmeasday** [comment]
@lorensr @stubailo FYI, in case you are interested!

File: imports/api/resolvers.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Lists } from './lists/lists';
+
+export default {
+  RootQuery: {
+    lists(_, { mine = false }, context) {
+      if (mine) {
+        // XXX: is this the correct way to do auth?
```
Comment (lorensr) [inline imports/api/resolvers.js]: Yep

Comment (tmeasday) [inline imports/api/resolvers.js]: I guess I meant returning an empty array rather than throwing some kind of exception. Is that what you meant?

Comment (lorensr) [inline imports/api/resolvers.js]: Oops nvm, I just meant trusting `context.userId`. I've also wondered what to do about errors

**Comment by hwillson** [comment]
Hi @tmeasday - what's the plan with this PR? Are you still planing on working on this or was the focus switched to GitHunt? I might be interested in continuing work on this; I know the GitHunt app is a much better full demonstration of Apollo's capabilities, but I think there is value in ramping Meteor users up on Apollo by leveraging the reference application most of them already know.

**Comment by tmeasday** [comment]
Hey @hwillson well the idea here was to help figure out what is going to be in Meteor 1.5 through figuring out what was difficult/impossible in this branch. 

I guess that is still the plan but I've been sidetracked for a while. I'm not sure when I'll get back to it; if you want take the branch further, I think it'll have value for people as an example of how to use Apollo w/ Meteor, especially pre-1.5. If that's something you want to work on and that's valuable for you, go for it!

**Comment by hwillson** [comment]
Great, thanks for the info @tmeasday - I'll chip away at the other outstanding items in this repo first, then look into this one a bit further.

**Comment by hwillson** [comment]
I'll close this for now - a lot has changed with Apollo over the past while, and more and more example apps showing how to integrate Meteor with Apollo have been popping up. I still think it would be great to have an Apollo branch on the `todos` app, but we'll leave it for another day. Thanks!

**Pull request closed without merging**